ui/src/message_popup/playtest/hmtca_scenario.jsx

```diff
@@ -0,0 +1,43 @@
+/* @flow weak */
+
+// This file defines the content for the HMTCA scenarios.
+// Note: This only supports text scenes right now.
+export type QuestionT = {
+  type:string, // Used as a label
+  text:string, // Plain question text
+  applesSceneNumber?:number, // Ask for text response and log it specially
+};
+
+
+// Different categories for classroom management
+const BUCKETS = [
+  { id: 201, text: 'Students refusing to work' },
+  { id: 202, text: 'Disrespect towards teachers' },
+  { id: 203, text: 'Disrespect towards females' },
+  { id: 204, text: 'Combination of the above' }
+];
+
+
+function slidesFor(cohortKey, bucketId) {
+  const slides:[QuestionT] = [];
+
+  if (bucketId === 201) {
```

Comment:
These are mostly placeholder scenes from now, but this shows how we can have different scenes based on the buckets they chose, and how to number them so the responses get tracked for reviewing later (eg., if you put a `note` or `open` or other type of question in these slides that's cool but they won't show up in the review at the end).